src/quo2/components/messages/system_message.cljs

```diff
@@ -0,0 +1,127 @@
+(ns quo2.components.messages.system-message
+  (:require [quo.react-native :as rn]
+            [quo.theme :as theme]
+            [quo2.components.buttons.button :as button]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo2.components.avatars.icon-avatar :as icon-avatar]))
+
+(def themes
+  {:light {:text       colors/black
+           :icon       colors/primary-50
+           :time       colors/neutral-50
+           :background colors/neutral-5}
+   :dark  {:text       colors/white
+           :icon       colors/primary-50
+           :time       colors/neutral-40
+           :background colors/neutral-95}})
+
+(defn get-color [key]
+  (get-in themes [(theme/get-theme) key]))
+
+(defn sm-timestamp [timestamp-str]
+  [rn/view {:margin-left 6}
+   [text/text {:size  :label
+               :style {:color          (get-color :time)
+                       :text-transform :none}} timestamp-str]])
+
+(defn sm-icon [icon]
+  [rn/view {:align-items  :center
+            :margin-right 6}
+   [icon-avatar/icon-avatar {:size  :medium
+                             :icon  icon
+                             :color :primary}]])
+
+(defmulti sm-render :type)
+
+(defmethod sm-render :deleted [_]
+  [rn/view {:align-items     :center
+            :justify-content :space-between
+            :flex            1
+            :flex-direction  :row}
+   [rn/view {:align-items    :center
+             :flex-direction :row}
+    [sm-icon :main-icons/placeholder20]
+    [text/text {:size  :paragraph-2
+                :style {:color        (get-color :text)
+                        :margin-right 5}} "Message deleted for everyone"]]
```

Comment:
Localize string by adding it in `en.json` file and fetch it with i18n package

Reply:
Done